endstate_rew/system.py

```diff
@@ -105,3 +105,46 @@ def initialize_simulation(molecule:Molecule, at_endstate:str='', platform:str='C
     #sim.context.setVelocitiesToTemperature(temperature)
     sim.context.setVelocities(_seed_velocities(sim, molecule))
     return sim
+
+#creating all ZINC systems
+
+base = '../data/hipen_data/'
```

Comment:
I think you can move `base` in the function scope. In that way it wouldn't interfere with any other variable named `base`.